local-development/.env

```diff
@@ -3,3 +3,4 @@ KEYCLOAK_REALM=fairspace
 WORKSPACE_CLIENT_ID=workspace-client
 WORKSPACE_URL=http://localhost:8080
 WORKSPACE_DEBUG_URL=http://localhost:3000
+DOCKER_LOGGING_DRIVER=json-file
```

Comment:
this is needed for mac users, linux users should use the default journald, so this shouldn't be added in the default configuration (only your local .env)

Reply:
Ah yes, I meant to leave it out of the commit. This should indeed not be added to the default configuration.